src/secrets.ts

```diff
@@ -9,8 +9,8 @@ export const LOGROCKET_APP_ID = process.env.REACT_APP_LOGROCKET_APP_ID;
 export const MIXPANEL_PROJECT_TOKEN =
   process.env.REACT_APP_MIXPANEL_PROJECT_TOKEN;
 
+// @debt WS_RELAY_URL is not used in code other the next one line
 export const WS_RELAY_URL = process.env.REACT_APP_WS_RELAY_URL;
```

Comment:
if so, can be removed too, I believe.

Reply:
removed @ 5a0e0729d2cdb8571b4f3c7bbc88cbedc7195eac